src/drafts/MarkdownViewer/_useListInteraction.ts

```diff
@@ -4,6 +4,17 @@ import {ListItem, listItemToString, parseListItem} from '../MarkdownEditor/_useL
 
 type TaskListItem = ListItem & {taskBox: '[ ]' | '[x]'}
 
+// Make check for code fences more robust per spec: https://github.github.com/gfm/#fenced-code-blocks
+const parseCodeFenceBegin = (line: string) => {
+  const match = line.match(/^ {0,3}(`{3,}|~{3,})[^`]*$/)
```

Comment:
```suggestion
  const match = line.match(/^ *(`{3,}|~{3,})[^`]*$/)
```

Looks like the spec says that an opening fence can be indented an indefinite number of spaces, but the closing fence can only be indented up to 3.

Reply:
The spec states that both an opening and closing fence can be indented no more than three spaces:

> A [fenced code block](https://github.github.com/gfm/#fenced-code-block) begins with a code fence, indented no more than three spaces.

> The closing code fence may be indented up to three spaces

I also tested and confirmed this is the case.

Reply:
Huh, the spec has an example with more than three: https://github.github.com/gfm/#example-104

But I also tested and confirmed, this doesn't work. So maybe this is a problem with our own pipeline. Guess this is all good then :+1:

Reply:
Side note, maybe we also should filter out lines starting with more than 3 spaces?

Ie, this is not a checklist item, but it's also not in a code block:

    - [ ] example

This came from this markdown:

```md
    - [ ] example
```